guides/common/modules/proc_configuring-satellite-for-outgoing-emails.adoc

```diff
@@ -55,11 +55,11 @@ The `SMTP username` and `SMTP password` specify the login credentials for the SM
 |====
 |Name| Example value
 |Delivery method    | Sendmail
-|Sendmail arguments | -i -t -G
+|Sendmail location | /usr/sbin/sendmail
+|Sendmail arguments | -i
 |====
 +
-The `Sendmail arguments` specify the options passed to the `sendmail` command.
-The default value is `-i -t`.
+For security reasons, both Sendmail location and Sendmail argument settings are read-only and can be only set via `/etc/foreman/settings.yaml` or `{foreman-installer}`.
```

Comment:
If you can provide me with installer arguments we are adding, I will add them here @ekohl

Reply:
Today we don't have these. It's limited to these:
https://github.com/theforeman/puppet-foreman/blob/8bfb8ce166fe8a92080bf69b56aa6e5f0e1c3047/manifests/init.pp#L21-L33

We should create a Redmine issue for this so it can be picked up and cherry picked if desired.

Reply:
Created https://projects.theforeman.org/issues/33543